target/linux/pistachio/config-4.4

```diff
@@ -319,6 +325,7 @@ CONFIG_LEGACY_PTYS=y
 CONFIG_LEGACY_PTY_COUNT=256
 CONFIG_LIBFDT=y
 CONFIG_LKDTM=y
+CONFIG_LOCKD=y
```

Comment:
After doing make kernel_menuconfig, CONFIG_LOCKD too getting removed.
This is dependent on CONFIG_NFS_FS which was enabled in config-4.1 originally but later on got removed but accidentally CONFIG_LOCKD option stayed in.